data/blank_mod/common/ideas/army_spirits.txt

```diff
@@ -347,6 +597,11 @@ ideas = {
 					factor = 2
 					has_tech = mobile_warfare
 				}
+
+				modifier = {
+					factor = 4
+					original_tag = GER
+				}
```

Comment:
This modifier is country-specific, please remove.

Reply:
Done.